api/src/test/java/org/openmrs/api/ObsServiceTest.java

```diff
@@ -540,6 +535,22 @@ public void getHandler_shouldHaveDefaultImageAndTextHandlersRegisteredBySpring()
 		Assert.assertNotNull(textHandler);
 	}
 	
+	@Test
+	@Verifies(value = "should purge complex obs when complex data file is missing from disk", method = "purgeObs(Obs)")
+	public void purgeComplexObs_shouldSucceedWhenCompleDataFileMissingFromDisk() throws Exception {
```

Comment:
dear @mks-d can you please update the test. it is written in the old style.
the https://wiki.openmrs.org/display/docs/Generate+Test+Case+Plugin can generate the test boilerplate according to the new style for you